keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -102,10 +102,35 @@ class ActuationPauser(
     mutableListOf<PersistentEvent>()
       .let { events ->
         events.addAll(originalEvents)
-        val relevantAppEvents = resourceRepository
-          .applicationEventHistory(resource.application, events.last().timestamp)
+
+        val oldestResourceEvent = events.last()
```

Comment:
watching you explaining it in the video -- I think the name here is a bit confusing since it's not the `oldest` event, it's actually `latest` event, meaning the last event for that application, correct?

Reply:
Nope, it really is the oldest. The list is ordered in descending order of timestamp, so the first is the newest, and the last is the oldest.

Reply:
isn't `events.last` returns the last publish event?

Reply:
No, that would be `events.first()` because the list is in descending order of timestamp, like I mentioned above.